test/integration/local/processing/get-completed-payment-requests.test.js

```diff
@@ -1,5 +1,8 @@
+const { SFI, CS } = require('../../../../app/constants/schemes')
 const db = require('../../../../app/data')
```

Comment:
I think these should be the other way around with a space between them

Reply:
The focus is get completed so wanted that separate

Reply:
Never mind, misunderstood.  Done!